src/pages/tag.html

```diff
@@ -0,0 +1,12 @@
+---
+title: Tags
+page-header: Tags
+---
+
+<div class="tag-container">
+	<div class="tag">Label</div>
+	<div class="tag cross-icon-tag">Label <span class="icon">x</span></div>
```

Comment:
`x` cross is optional for each type of tags, `icon` should be `close`, x in designs is build from `close` tabler icon, not "x" letter